### DISCUSSION TITLE
Auto update but use built in theme

### DISCUSSION THREAD

**Discussion by cprivitere** [JanDeDobbeleer/oh-my-posh#5382] [Troubleshoot] [answered]
Hi, I want to use the built in powerlevel10k_classic theme but also enable auto upgrades of oh-my-posh. 

**Why?**
I'd like automatic upgrades of oh-my-posh as well as taking any upgrades or tweaks to the powerlevel10k_classic theme that are made. 

**Why I'm stuck**
I don't see how to enable auto upgrades without editing the theme file itself which means I'm no longer using the bundled and thus managed/upgraded by oh-my-posh theme file. 

Is there a way to achieve this?

**Comment by JanDeDobbeleer** [comment, marked as answer]
@cprivitere this is a feature I still need to develop. That said, the themes aren't really updated that much anymore so it's highly likely it stays the same for a while.
